orbit/cmd/desktop/desktop_linux.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"slices"
+
+	"github.com/godbus/dbus/v5"
+	"github.com/rs/zerolog/log"
+)
+
+//go:embed icon_dark.png
+var iconDark []byte
+
+func blockWaitForStopEvent(_ string) error {
+	log.Debug().Msg("communication channel helpers are not implemented for this platform")
+	return nil
+}
+
+func trayIconExists() bool {
+	conn, err := dbus.SessionBus()
+	if err != nil {
+		log.Error().Err(err)
+	}
+
+	// Get the name we would expect systray to reserve for our tray icon.
+	trayIconDbusName := fmt.Sprintf("org.kde.StatusNotifierItem-%d-1", os.Getpid())
```

Comment:
Curious why is it KDE? (Gnome)

Reply:
Just convention I think.  It's an arbitrary naming scheme, but I see it in a lot of examples, and it's what [systray uses](https://github.com/fyne-io/systray/blob/master/systray_unix.go#L182).